src/app/devices/event-log/event-log.component.spec.ts

```diff
@@ -17,34 +16,34 @@ describe('EventLogComponent', () => {
   let snackBarSpy: jasmine.SpyObj<MatSnackBar>
 
   // Sample responses for cloud and non-cloud modes
-  const mockData = {
-    records: [
-      {
-        DeviceAddress: 255,
-        EventSensorType: 15,
-        EventType: 1,
-        EventOffset: 2,
-        EventSourceType: 104,
-        EventSeverity: 8,
-        SensorNumber: 255,
-        Entity: 34,
-        EntityInstance: 0,
-        EventData: [
-          64,
-          19,
-          0,
-          0,
-          0,
-          0,
-          0,
-          0
-        ],
-        EntityStr: 'BIOS',
-        Desc: 'Cloud Event'
-      }
-    ],
-    hasMoreRecords: false
-  }
+  // const mockData = {
```

Comment:
Do we need this commented code?

Reply:
i was thinking we would bring these tests back